cfgov/cfgov/urls.py

```diff
@@ -413,7 +415,8 @@
     except ImportError:
         pass
 
-if settings.DEPLOY_ENVIRONMENT != 'build':
+
+if settings.DEPLOY_ENVIRONMENT not in ['build', 'beta']:
```

Comment:
Because we add `r'^(?i)askcfpb/'` as [a flagged URL below](https://github.com/cfpb/cfgov-refresh/pull/2998/files#diff-e45111e649c8f0371d5367b7351f1dd8R433), this doesn't actually seem to work. It'll need to be a fallback for that flagged URL.